tests/fsharp/Compiler/Conformance/DataExpressions/ComputationExpressions.fs

```diff
@@ -720,7 +720,7 @@ type ContentBuilder() =
     member this.Run(c: Content) =
         let crlf = "\r\n"B
         [|for part in List.rev c do
-            yield! part.Array.[part.Offset..(part.Count+part.Offset-1)]
+            yield! part.Array[part.Offset..(part.Count+part.Offset-1)]
             yield! crlf |]
```

Comment:
Do we not also value keeping expr.[] tests working as well as expr[] tests until we fully deprecate expr.[]

Reply:
We do but we do have a lot of those tests in various places.  Given the nature of this test (primarily about computation expressions) it feels we could use either, it would be wrong to duplicate out such a large test for this one case.   I think the use of `Array[]` removed the FCS report of the hidden informational diagnostic, which - given that it's unrelated - seems to be better for the test.